receiver/vcenterreceiver/internal/mockserver/client_mock.go

```diff
@@ -45,8 +45,8 @@ type soapEnvelope struct {
 }
 
 // MockServer has access to recorded SOAP responses and will serve them over http based off the scraper's API calls
-func MockServer(t *testing.T) *httptest.Server {
-	vsphereMock := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+func MockServer(t *testing.T, useTls bool) *httptest.Server {
```

Comment:
```suggestion
func MockServer(t *testing.T, useTLS bool) *httptest.Server {
```

linter wants this
> internal/mockserver/client_mock.go:48:31: var-naming: func parameter useTls should be useTLS (revive)
func MockServer(t *testing.T, useTls bool) *httptest.Server {